packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/imageEditors/Resizer.ts

```diff
@@ -108,59 +110,117 @@ export function doubleCheckResize(
+        return;
+    }
     const result: CreateElementData[] = [];
     Xs.forEach(x =>
         Ys.forEach(y =>
             result.push(
-                (x == '') != (y == '') ? getResizeHandleHTML(x, y, resizeBorderColor) : null
+                (x == '') != (y == '')
+                    ? getResizeHandleHTML(
+                          x,
+                          y,
+                          resizeBorderColor,
+                          circularHandlers,
+                          true /** isSideHandlers */
+                      )
+                    : null
             )
         )
     );
     return result;
 }
 
-function getResizeHandleHTML(x: X, y: Y, borderColor: string): CreateElementData {
+function getResizeHandleHTML(
+    x: X,
+    y: Y,
+    borderColor: string,
+    circularHandlers?: boolean,
+    isSideHandlers?: boolean
+): CreateElementData {
     const leftOrRight = x == 'w' ? 'left' : 'right';
     const topOrBottom = y == 'n' ? 'top' : 'bottom';
     const leftOrRightValue = x == '' ? '50%' : '0px';
     const topOrBottomValue = y == '' ? '50%' : '0px';
     const direction = y + x;
-
     return x == '' && y == ''
         ? {
               tag: 'div',
-              style: `position:absolute;left:0;right:0;top:0;bottom:0;border:solid 1px ${borderColor};pointer-events:none`,
+              style: `position:absolute;left:0;right:0;top:0;bottom:0;border:solid 2px ${borderColor};pointer-events:none;`,
           }
         : {
               tag: 'div',
               style: `position:absolute;${leftOrRight}:${leftOrRightValue};${topOrBottom}:${topOrBottomValue}`,
               children: [
                   {
                       tag: 'div',
-                      style: `position:relative;width:${RESIZE_HANDLE_SIZE}px;height:${RESIZE_HANDLE_SIZE}px;background-color: ${borderColor};cursor:${direction}-resize;${topOrBottom}:-${RESIZE_HANDLE_MARGIN}px;${leftOrRight}:-${RESIZE_HANDLE_MARGIN}px`,
+                      style: setHandlerStyle(
+                          circularHandlers,
+                          isSideHandlers,
+                          y,
+                          borderColor,
+                          direction,
+                          topOrBottom,
+                          leftOrRight
+                      ),
                       className: ImageEditElementClass.ResizeHandle,
                       dataset: { x, y },
                   },
               ],
           };
 }
+
+function setHandlerStyle(
+    circularHandlers: boolean,
+    isSideHandlers: boolean,
+    y: string,
+    borderColor: string,
+    direction: string,
+    topOrBottom: string,
+    leftOrRight: string
+) {
+    if (!circularHandlers) {
+        return `position:relative;width:${RESIZE_HANDLE_SIZE}px;height:${RESIZE_HANDLE_SIZE}px;background-color: ${borderColor};cursor:${direction}-resize;${topOrBottom}:-${RESIZE_HANDLE_MARGIN}px;${leftOrRight}:-${RESIZE_HANDLE_MARGIN}px;`;
+    } else if (!isSideHandlers) {
```

Comment:
do not use the combination of boolean values to decide which value to return. It is very hard to understand. Use const enum instead.